### PR TITLE
Feat/week 5 6 js

### PR DESCRIPTION
**Pull request by ibraelakhdar**

### CODE REVIEW

File: Pages/cours.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <title>COURS</title>
     <link rel="stylesheet" href="./../CSS/bootstrap.min.css">
     <link rel="stylesheet" href="./../CSS/cours.css">
+    <script text="text/javascript" src="./cours.js"></script>
```
Comment (ibraelakhdar) [inline Pages/cours.html:9]: you should create a scripts directory

File: Pages/cours.html [hunk 2]
```diff
@@ -21,7 +22,7 @@
         </div>
         <div class="card-body">
             <h3 class="text-center">Administrer les cours</h3>
-            <table class="table table-dark">
+            <table id="classTab" class="table table-dark">
```
Comment (ibraelakhdar) [inline Pages/cours.html:25]: classes_table would be better

File: Pages/cours.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        "label": "Base de données",
+        "period": "01/02 - 15/03/2020",
+        "nbHour": 40+" h",
+        "teacher": "Nom Prénom",
+        "detail": "Morem ipsum dolor sit amet, consectetur adipisicing elit. Amet numquam aspernatur!"
+    }
+]
+
+//#region Iniatilize page
+function getClasses() {
```
Comment (ibraelakhdar) [inline Pages/cours.js:28]: displayClasses would be correct since that's what the function does

File: Pages/cours.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
+var classes = [
+    {
+        "id": 1,
+        "label": "Algorithmes",
+        "period":"01/01 - 01/03/2020",
+        "nbHour": 35+" h",
```
Comment (ibraelakhdar) [inline Pages/cours.js:6]: A better usage, would be either let the nbHour field a string and then in the function add we split it to 2 terms "35" and "h" or create an object with a unit and value which is better

File: Pages/cours.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    const addClass = {
+        "id": classes.length+1,
+        "label": title,
+        "period":period,
+        "nbHour": nbHour,
+        "teacher": teacher,
+        "detail": description
+    };
+    classes.push(addClass);
+    document.getElementById("titleInput").value = "";
```
Comment (ibraelakhdar) [inline Pages/cours.js:94]: you can create a function that reinit fields (move all the reinit instructions into one function and call it here)

File: Pages/cours.js (hunk 4 above)
Comment (DevGaby) [reply Pages/cours.js:6]: je rajoute le 'h' dans le html alors?

**Pull request merged**